src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__content-wrapper">
+        <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+        class="card__img"
+        >
+        <h5 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h5>
+        <p class="card__code">
+          Product code: 195434
+        </p>
+        <div class="card__review-container">
+          <ul class="card__stars stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-p">Price:</p>
```

Comment:
what does `card__price-p` mean? let's give more descriptive class for this.

Reply:
i thought it was okay to use tag-names like that, like 'card__price-span' etc since its pretty understandable what it means 🙂
p as a paragraph, but okay I'll change it

Reply:
@polinavafik `"p"` can also be an abbreviation for `"padding"`, which will often be found in CSS frameworks, so you should definitely not shorten class names to unreadable ones